core/src/core-api/index.js

```diff
@@ -1,11 +1,12 @@
 import { config } from './config';
 import { auth } from './auth';
+import { elements } from './dom-elements';
 
 export const LuigiConfig = config;
 export const LuigiAuth = auth;
+export const LuigiElements = elements;
 
 // Expose it window for user app to call Luigi.setConfig()
-window.Luigi = Object.assign(
-  config,
-  auth
-);
+window.Luigi = config;
+window.Luigi.auth = auth;
```

Comment:
We implemented it like this, but we should have a talk with @hardl to align the window.Luigi API.
Some possibilities:
```
Luigi.auth
Luigi.auth()
Luigi.authManager()
```
Since config, auth and elements are instances of their classes, it is not possible to assign all of them at the root level of Luigi.

Reply:
Done